org.eclipse.xtext/src/org/eclipse/xtext/naming/QualifiedNameInterning.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2022 itemis AG (http://www.itemis.eu) and others.
```

Comment:
The copyright header is probably not the correct one.

Reply:
Don't know... I just left it as-is.